CHANGELOG.md

```diff
@@ -13,10 +13,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Changed
 - Structure Block Registry Processor can now be used for data generated structures through a processor_list
+- Adjusted the hitbox size for Lotus Flowers
+- The Lotus Time Capsule now spawns a Lotus Flower when placed
+- Lotus Time Capsule blocks now remove any Lotus Flowers above them when destroyed
+
+### Fixed
+- The Umbral Infiltrator no longer jitters when thrown by a void player
```

Comment:
Was this also a problem in 1.20.1?

Reply:
Sure is. I got on the most recent public build to test it out and it appears to be the case.